.github/workflows/update-clair.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    tags:
+      - v*.*.*
```

Comment:
```suggestion
      - v1.*.*
```

We're going to rev this at some point.

Reply:
Not sure what the point in limiting to v1 tags is? Would we not want this behavior when it's iterated to v2 v3 etc (assuming we don't have a better mechanism by then)?

Reply:
I suppose I just thought it'd create PRs for everything and we'd just close what we didn't want. What would be the advantage of paring down?

Reply:
Yeah, fair enough.